lago/plugins/vm.py

```diff
@@ -411,22 +404,35 @@ def copy_to(self, local_path, remote_path, recursive=True):
         with LogTask(
             'Copy %s to %s:%s' % (local_path, self.name(), remote_path),
         ):
-            with self._scp() as scp:
-                scp.put(
-                    files=local_path,
-                    remote_path=remote_path,
-                    recursive=recursive,
-                )
+            try:
+                with self._scp() as scp:
+                    scp.put(
+                        files=local_path,
+                        remote_path=remote_path,
+                        recursive=recursive,
+                    )
+            except OSError as err:
```

Comment:
What can raise `OSError`? Are you sure that the scp module doesn't reaise `SCPException`?